packages/template-retail-react-app/app/hooks/use-auth-modal.js

```diff
@@ -162,6 +174,9 @@ export const AuthModal = ({
                         message: formatMessage(API_ERROR_MESSAGE)
                     })
                 }
+            },
+            email: async (data) => {
+                // Handle resend passwordless email logic here
```

Comment:
Hey @yunakim714 wondering if we should consider naming this `passwordless` instead to be more specific since the reset password flow will also be sending an email.
```suggestion
            passwordless: async (data) => {
                // Handle resend passwordless email logic here
```